util/geoJsonCenter.js

```diff
@@ -16,4 +20,4 @@ module.exports = function( geometry ){
   }
 
   return;
```

Comment:
`return;` is equivalent to `return undefined;`, why are `return false;` or `return null;` better?